shared/constants/platform-specific/index.native.tsx

```diff
@@ -137,14 +138,17 @@ export const showShareActionSheet = async (options: {
 // TODO rewrite this, v slow
 const loadStartupDetails = async () => {
   const [routeState, initialUrl, push, share] = await Promise.all([
-    Container.neverThrowPromiseFunc(async () =>
```

Comment:
So there was a long standing race here. In this call we make an rpc but we try to do this as early as possible. With the new code flows its early enough to be before the onConnect call happens so this never resolves. We actually already plumb this value as a constant so we can get it sync and act on it immediately so just do that